src/Microsoft.Diagnostics.EventFlow.Consumers.EventContextConsoleApp/config.json

```diff
@@ -0,0 +1,39 @@
+﻿{
+  "healthReporter": {
+    "type": "CsvHealthReporter",
+    "logFileFolder": ".",
+    "logFilePrefix": "HealthReport",
+    "minReportLevel": "Warning",
+    "throttlingPeriodMsec": "1000"
+  },
+  "inputs": [
+    {
+      "type": "Trace",
+      "traceLevel": "Warning"
+    }
+  ],
+  "filters": [
+  ],
+  "outputs": [
+    {
+      "type": "ApplicationInsights",
+      "instrumentationKey": "***REMOVED***"
+    },
+    {
+      "type": "StdOutput"
+    },
+    {
```

Comment:
It was needed to check how context is serialized to other outputs. However as a sample it does not need to write to all outputs and I left StdOut and AI only